### ISSUE TITLE
Keyboard navigation (Page Up/Down, Space) does not work until page is focused by clicking

### ISSUE THREAD

**Issue by smillardHNTB** [smartcitiesdata/discovery_ui#200]
Mac - Chrome 76.0.3809.132

Keyboard navigation (Page Up, Page Down, space bar) does not work until you click somewhere on the page.  This happens on while filtering on facets also.
